README.md

```diff
@@ -4,7 +4,7 @@ This is an exercise in TDD-ing and OO Design. It's broken into 7 features:
 
 ## Features
 
-1. When you run ruby `tic_tac_toe/game_runner.rb`, an empty grid is rendered on the screen.
+1. When you run ruby `oo-tic-tac-toe/game_runner.rb`, an empty grid is rendered on the screen.
```

Comment:
@orhanna14 this is a specification. The script that runs the app separate from the code that implements the game. I don't think you created that script.

I'm not sure the original location was the best. What I would love instead is  to run it without Ruby because that is in the shebang line of the script and the file is executable. In the meantime the specification could be changed to `ruby ./game_runner.rb` so that the file was in the root directory.